packages/react-card/Spec.md

```diff
@@ -0,0 +1,432 @@
+| header      | _slot_ | undefined | Header of the card                                |
+| description | _slot_ | undefined | Further description of the content                |
+
+### CardBody
+
+This component has no custom props and only ensures the proper positioning of content and itself within the card.
+
+### CardPreview
+
+![visual anatomy of the CardPreview component](./assets/CardPreview.png)
+
+#### API
+
+| Property | Values | Default   | Purpose                                                  |
+| -------- | ------ | --------- | -------------------------------------------------------- |
+| src      | string | undefined | Source of the media to be displayed on the `img` element |
+| alt      | string | undefined | Alternative text to describe the media presented         |
+| logo     | _slot_ | undefined | A logo/file type to add over the image preview           |
+
+### CardFooter
+
+This component has no custom props and only ensures the proper positioning of content and itself within the card.
+
+## Sample Code
+
+### Compact
+
+```jsx
+<Card orientation="horizontal">
+  <CardHeader image={<img src="powerpoint.png" />} header="Team offsite 2020" description="Onedrive > Files" />
+</Card>
+```
+
+### App
+
+```jsx
+<Card>
+  <CardHeader image={<img src="app_logo.png" />} header="App Name" description="Developer" />
+  <CardBody>
+    Donut chocolate bar oat cake. Dragée tiramisu lollipop bear claw. Marshmallow pastry jujubes toffee sugar plum.
+  </CardBody>
+</Card>
+```
+
+### Task
+
+```jsx
+<Card>
+  <CardHeader header="Alert in Teams when a new document is uploaded in channel" description="By Microsoft" />
+  <CardFooter>
+    <span>Automated</span>
+    <span>3290 uses</span>
+  </CardFooter>
+</Card>
+```
+
+### Image
+
+```jsx
+<Card>
+  <CardPreview src="monkey.png" alt="image of monkey" />
+  <CardBody>
+    <div>
+      <span>Length</span>
+      <span>4.6-39.6 IN</span>
+      <span>11.7-100.6 cm</span>
+    </div>
+    <div>
+      <span>Diet</span>
+      <span>Omnivore</span>
+      <span>nuts, fruits, eggs</span>
+    </div>
+  </CardBody>
+  <CardFooter>
+    <Button>Share</Button>
+  </CardFooter>
+</Card>
+```
+
+## Behaviours
+
+### Mouse
+
+### Interactive cards
+
+![Visual interaction of interactive cards using the mouse](./assets/interactive-interaction-mouse.png)
+
+The card surface can be used as a clickable target
+
+1. Rest
+
+2. OnHover
+
+   Cursor style changes to pointer, background color & elevation values change to show card is an actionable element
+
+3. OnClick
+
+   Executes action assigned to card i.e. “Open file, Open link in browser etc...”
+
+#### Selectable cards
+
+![Visual interaction of selectable cards using the mouse](./assets/selectable-interaction-mouse.png)
+
+Clicking the card surface can toggle the checkbox state to `selected`. Note that checkboxes are composed together as an additional component and are not provided by default.
```

Comment:
> checkboxes are composed together as an additional component and are not provided by default

What does this mean ? can you add some sample code for this scenario ?

Reply:
This was leftover from the design spec, I'll remove it. Context here is that the Checkbox is not Card implementation but instead it's own component, I think.